CITATION.cff

```diff
@@ -0,0 +1,159 @@
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "BIDS Stats Models Specification"
+
+version: 0.1.0
+
+abstract: BIDS Stats Models Specification.
+
+message: "If you are using the BIDS stats model, please cite it as below."
+
+repository-code: "https://github.com/bids-standard/stats-models.git"
+
+# identifiers:
+#   - description: This is the collection of archived snapshots of all releases of BIDSpm
+#     type: doi
+#     value: "10.5281/zenodo.3554331"
+
+contact:
+  - family-names: XXX
+    affiliation: XXX
+    given-names: XXX
+    email: XXX@XXX.XXX
```

Comment:
@effigies should this be you?

Reply:
Does this just have to be one? If so, me or Alejandro is fine. If we can have both, that makes sense, IMO.

Reply:
Just checked the schema.
This is actually not required.
And it can have several entries.

Should I remove it or add you both?